src/Middleware/StaticFiles/src/FileExtensionContentTypeProvider.cs

```diff
@@ -461,7 +461,7 @@ public bool TryGetContentType(string subpath, [MaybeNullWhen(false)] out string
                 return null;
             }
 
-            return path.Substring(index);
+            return path[index..];
```

Comment:
This also seems to be primarily a style change?

Reply:
This is a range operator, rather than using substring. Under the hood, it uses `Span<T>`. 
More info - https://docs.microsoft.com/en-us/dotnet/csharp/tutorials/ranges-indexes

Reply:
Sure, but the method's return value is still a string so the allocation still happens anyways.

Reply:
Ohh OK. You might be correct. I will close this PR now! As there are no major changes left in it.